src/components/AnswerSection.tsx

```diff
@@ -217,7 +230,7 @@ const AnswerSection = ({
         <Actions
           tryLater={tryLater}
           giveUp={async () => {
-            await storeResult('incorrect')
+            await storeResult(secondAttempt ? 'similar' : 'incorrect')
```

Comment:
Why is an answer similar just because it is the second attempt?

Reply:
You only have a second attempt, if your first input was 'similar'.

Reply:
Sounds weird :see_no_evil: What is the reason behind this?

Reply:
If the correct answer is "Der Hammer" and you enter "Das Hammer" or "Der Hamer" you get an second attempt, because you are really close. If you enter "Der Meisel" you just get the correct answer and you input is counted as 'incorrect'. If you try and it is similar, and then give up, you solution is still counted as 'similar'. It is an design question, if you disagree or have other better ideas about it, we should discuss it on tuesday evening.

Reply:
No, just did not make sense for me at first, but with your explanation it does, thanks :+1: